src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/label.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="thibault" id="label">
+        <createTable tableName="LABEL">
```

Comment:
I do it by intuition but I think it goes something like this:

- **don't expose autoincrement IDs to the user**. because that allows the user to gain information about what's going on inside the db. if you have id 1234 you know that there exist ids 0-1233 also; if you have an endpoint `GET /thing/<id>` with a security hole you can now start trawling IDs
- **don't rely on externally generated ids to be unique**. even if they're uuids they might not be one day; it requires a little too much trust in someone else's implementation for my liking
- **you only have to worry about this for single-column database PKs**. this is the most squishy and hard to get "the thing in my head" into words. but in the case of labels you're looking up by >1 field and one of them is a FK that is hidden from the user. there's no GET (clusterid, label) endpoint to attack with an incrementing id, labels are (kinda) unguessable, and getting a label value doesn't give you a ton of information anyway.